src/thunderbots/software/radio_communication/mrf_backend.h

```diff
@@ -49,8 +54,23 @@ class MRFBackend
      */
     void send_vision_packet();
 
+    /**
+     * Parses a robot status message from the dongle, and publishes it to the ROS topic.
+     * 
+     * @param index robot number
+     * @param data the message data
+     * @param len the length of the message
+     * @param lqi link quality
```

Comment:
To clarify, does LQI work like signal bars on a phone, or does it have a unit?

Reply:
Oops, sorry for the late reply, I think RSSI is more related to signal bar strength? I can guess link quality is a function of RSSI, packet drop rate (maybe?), and others.